api_tests/lib_sdk/utils.ts

```diff
@@ -16,3 +20,66 @@ export async function timeout<T>(ms: number, promise: Promise<T>): Promise<T> {
     // Returns a race between our timeout and the passed in promise
     return Promise.race([promise, timeout]);
 }
+
+export const DEFAULT_ALPHA = {
```

Comment:
Hmm, that is exactly the kind of code we wanted to avoid / abstract over with the new testing framework. Unfortunately, we can't reuse the other part well as it is coupled with starting up the ledgers etc.

I guess this should be fine for now as it helps us move forward with the e2e tests but it still bugs me :D 

Can we not just merge all the tests into one file, then we don't have to create something global here that looks like it is supposed to be used across tests.
That's what bugs me: these kind of globals create coupling between code snippets where there shouldn't or doesn't have to be coupling. A variable scoped to a single test file at least doesn't invite re-use in other tests :)